content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Istio Webhook Management [Experimental]
+description: How to manage webhooks in Istio through istioctl.
+weight: 100
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks: Galley and the sidecar injector. By default,
+these webhooks manage their own configurations. From a
+security perspective, this default behavior is not recommended because a compromised webhook could then conduct
+privilege escalation attacks.
+
+This task shows how to use the new [{{< istioctl >}} x post-install webhook](/docs/reference/commands/istioctl/#istioctl-experimental-post-install-webhook) command to
+securely manage the configurations of the webhooks.
+
+## Getting started
+
+* Install Istio with [DNS certificates configured](/docs/tasks/security/dns-cert) and
+`global.operatorManageWebhooks` set to `true`.
+
+    {{< text bash >}}
+    $ cat <<EOF | istioctl manifest apply -f -
+    apiVersion: install.istio.io/v1alpha2
+    kind: IstioControlPlane
+    spec:
+      values:
+        global:
+          operatorManageWebhooks: true
+          certificates:
+            - secretName: dns.istio-galley-service-account
+              dnsNames: [istio-galley.istio-system.svc, istio-galley.istio-system]
+            - secretName: dns.istio-sidecar-injector-service-account
+              dnsNames: [istio-sidecar-injector.istio-system.svc, istio-sidecar-injector.istio-system]
+    EOF
+    {{< /text >}}
+
+* Install [`jq`](https://stedolan.github.io/jq/) for JSON parsing.
+
+## Check webhook certificates
+
+To display the DNS names in the webhook certificates of Galley and the sidecar injector, you need to get the secret
+from Kubernetes, parse it, decode it, and view the text output with the following commands:
+
+{{< text bash >}}
+$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+$ kubectl get secret dns.istio-sidecar-injector-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+{{< /text >}}
+
+The output from the above commands should include the DNS names of Galley and the sidecar injector, respectively:
+
+{{< text plain >}}
+X509v3 Subject Alternative Name:
+  DNS:istio-galley.istio-system.svc, DNS:istio-galley.istio-system
+{{< /text >}}
+
+{{< text plain >}}
+X509v3 Subject Alternative Name:
+  DNS:istio-sidecar-injector.istio-system.svc, DNS:istio-sidecar-injector.istio-system
+{{< /text >}}
+
+## Enable webhook configurations
+
+1.  To generate the `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` configuration files, run the following
+command.
+
+    {{< text bash >}}
+    $ istioctl manifest generate > istio.yaml
+    {{< /text >}}
+
+<!-- TODO (lei-tang): improve the UX for obtain MutatingWebhookConfiguration -->
```

Comment:
Remove this line. It is causing the numbered list not to render properly.

Reply:
Done, thanks!